ruby-challenges.rb

```diff
@@ -18,13 +30,50 @@
 
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
+def num(array)
+    array.inject(0, :+)
+end
+p num(nums_array1)
+p num(nums_array2)
```

Comment:
Newer versions of Ruby also have another built-in method called sum that does the same thing.  Worth looking into the difference between those two methods.